LRQA-29315, LRQA-29421 | 7.0.x

GinsonRen opened the pull request:
https://issues.liferay.com/browse/LRQA-29315
https://issues.liferay.com/browse/LRQA-29421

brianwulbern commented:
SF for JPince.

* Changed the localized integers so that they wouldn't have the same 2, 1, 3 order that the English integers had, so we can assert that changing the language in fact updates the order of assets by ascending integer.

brianwulbern commented:
Pushed to BChiu.
https://github.com/brianchiu/liferay-portal/pull/965

Ithildir committed:
LPS-69470 Add option to force a calculated Bundle-Version

Hanlf committed:
LRQA-29315 Add multiple field sets before choosing it

Ithildir committed:
LPS-66762 Fix gradle-plugins-baseline's smoke test

We need to have sample-100 installed in the local Maven cache even before the configuration phase of the "baseline" task.

Ithildir committed:
LPS-69470 Use latest ant-bnd

Ithildir committed:
LPS-69470 Add option to Gradle plugin

Ithildir committed:
LPS-69470 Use latest gradle-plugins-baseline

Ithildir committed:
LPS-69470 Force calculated Bundle-Version for Liferay modules

Ithildir committed:
LPS-69470 Semver gradle-plugins-baseline

Ithildir committed:
LPS-69470 Update readme

Ithildir committed:
LPS-69470 Update changelog

Ithildir committed:
LPS-69473 Reuse method

Ithildir committed:
LPS-69473 Automatically add Maven Central as repository

Ithildir committed:
LPS-69473 Update changelog

brianchandotcom committed:
LPS-69470 prep next, use latest

brianchandotcom committed:
LPS-69470 update jar

brianchandotcom committed:
LPS-69470 cache

brianchandotcom committed:
LPS-69473 prep next, use latest

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/static/portal-osgi-web/.gitrepo'.

liuxingye committed:
LRQA-29421 Use click for page transitions

liferay-continuous-integration commented:
<h1>Some tests FAILED.</h1><p>Build Time: 1 hour 4 minutes 40 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/7.0.x">7.0.x</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/7e28dcd9b1469744fca1e254eaf44763cff463de">7e28dcd9b1469744fca1e254eaf44763cff463de</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest(7.0.x)/72/">test-portal-acceptance-pullrequest(7.0.x)</a></strong></strike></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2437/">test-portal-acceptance-pullrequest-batch(7.0.x)/compile-jsp-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2901/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2743/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat8-mariadb100-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2778/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2817/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat8-oracle12-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1127/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2596/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-tomcat8-sybase16-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2779/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2902/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-smoke-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2744/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-hypersonic20-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2438/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2740/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2439/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/10</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2849/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/11</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2421/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/12</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2819/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/13</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2819/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/14</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1116/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/15</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2495/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/16</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2746/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/17</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1182/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/18</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2771/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/19</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2818/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/2</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2440/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/20</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1129/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/21</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2598/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/22</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2904/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/23</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2868/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/24</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2643/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/25</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2781/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/26</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2742/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/27</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2820/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/28</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2422/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/29</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2818/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/3</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1117/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/30</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2869/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/31</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2782/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/32</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2644/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/33</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2820/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/34</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2821/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/35</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2615/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/36</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2744/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/37</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2743/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/38</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1118/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/39</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1128/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/4</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2423/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/40</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2780/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/41</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2822/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/42</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2821/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/43</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2616/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/44</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2870/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/45</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2903/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/5</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2780/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/6</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2597/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2745/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2741/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-tomcat8-mysql56-jdk8/9</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/1120/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-mariadb100-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2871/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-mariadb100-jdk8/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2783/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-mysql56-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2745/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-mysql56-jdk8/1</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2823/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-oracle12-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2781/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-oracle12-jdk8/1</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2872/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-postgresql94-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2824/">test-portal-acceptance-pullrequest-batch(7.0.x)/functional-upgrade-tomcat8-postgresql94-jdk8/1</a></li><li><strike><strong><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2784/">test-portal-acceptance-pullrequest-batch(7.0.x)/portal-web-jdk8</a></strong></strike></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-dist(7.0.x)/22/">test-portal-acceptance-pullrequest-dist(7.0.x)/jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-source(7.0.x)/56/">test-portal-acceptance-pullrequest-source(7.0.x)</a></li></ul><h5>For more details click <a href="https://test-1-1.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(7.0.x)/builds/72/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest(7.0.x)/72/">test-portal-acceptance-pullrequest(7.0.x)</a></h5><h6>Job Results:</h6><p>66 Jobs Passed.<br />2 Jobs Failed.</p><pre><code>Completed with the status of FAILURE.</code></pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(7.0.x)/2784/">test-portal-acceptance-pullrequest-batch(7.0.x)/portal-web-jdk8</a></h5><h6>Job Results:</h6><p>56 Tests Passed.<br />1 Test Failed.</p><ol><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch%287.0.x%29/AXIS_VARIABLE=0,label_exp=!master/2784//testReport/com.liferay.source.formatter/SourceFormatterTest/testSourceFormatter">SourceFormatterTest.testSourceFormatter</a></li></ol></li></ol><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(7.0.x)/">here</a>.</h5>

brianwulbern commented:
SF for @liuxingye @GinsonRen 

* Removed the unnecessary _value1_ parameter from the Click line, as it doesn't need to find any text.

brianwulbern commented:
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/45231
https://github.com/brianchandotcom/liferay-portal/pull/45232 (only 29421 on master)
